### PR TITLE
Update README.md with PyPI restriction

### LINKED ISSUE #16: Docu: PyPi restriction
Please be clear in your documentation (e.g. README) that your package does check the "latest version" from PyPi only. PyPi is not the whole universe and also extrem uncommon in the current days because of security issues.

The point is that there are much more ways to distribute packages then PyPi. You don't have to support all these ways but you should clear about that you serve PyPi only.

### PR DESCRIPTION
**Pull request by buhtz**
My proposal for #16.

Fix #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ To install:
 
 This will:
 
-- Show a warning if the given version is not the latest. The warning includes the package name, the given version, and the latest version.
+- Show a warning if the given version is not the latest compared to the the latest version available on [PyPI](https://pypi.org) package index. The warning includes the package name, the given version, and the latest version.
```
Comment (alexmojaki) [inline README.md]: "PyPI package index" is RAS syndrome.

**Comment by alexmojaki** [comment]
PyPI is already mentioned repeatedly and visibly in the README, including in the first sentence and very close to your suggested change. This wording doesn't feel like an improvement.

File: README.md (hunk 1 above)
Comment (buhtz) [reply README.md]: Don't know the term "RAS syndrom".

**Comment by buhtz** [comment]
> PyPI is already mentioned repeatedly and visibly in the README

The bullet point list do describe the behavior of your package. But in the current state the description do suggest things that are not real. The behavior description should be more specific.

**Pull request closed without merging**